ssp

```diff
@@ -745,10 +745,7 @@ sub get_local_ipaddrs_aref {
     }
     for my $line (@output) {
         if ( $line =~ m{ (\d+\.\d+\.\d+\.\d+) }xms ) {
```

Comment:
On thing I can think of here is to expand this regex - but there really is no value in doing so.
Example:
```
         if ( $line =~ m{ (\d+\.\d+\.\d+\.\d+) }xms ) { 
```
could be
```
         if ( $line =~ m{ (([0-255]\.{3}[0-255])) }xms ) { 
```
Only concern there rises out of the push on Line 748 being that at that point $1 is assumed to be.  

I don't think that's even a fail point.  Passing as is!

Reply:
I like your regex better, though I hadn't thought of updating that regex at the time I touched the code.

Pushing $1 on line 748 should also be safe from any 'undefined value' warnings because if it's not defined (no regex match), the if() won't be true and it will never run that block of code, if that's what you mean?  Sorry if I misunderstood that part.

Reply:
Yea I think you're right.  cool!  I didn't test that regex but I was just trying to emphasize the checking to whatever end.